.circleci/config.yml

```diff
@@ -618,6 +618,7 @@ workflows:
   check_buildtype_device:
     jobs:
       - detekt
+      - firebase_screenshots
```

Comment:
Please remove it?

Reply:
yes, as the screenshots are taken we can remove it :)

Reply:
But the option to run the screenshots on firebase without modifying the config.yml file for certain branches/PRs would be nice. Any ideas about how we could accomplish this?

Reply:
done

Reply:
It is a nice idea, I don't have any on top of my head. if anyone has ideas let me know?